src/effects/perform.rs

```diff
@@ -90,6 +90,24 @@ fn perform_toggle_layer_alias(ktrl: &mut Ktrl, name: String, value: KeyValue) ->
     Ok(())
 }
 
+fn perform_toggle_profile(ktrl: &mut Ktrl, name: String, value: KeyValue, on: bool) -> Result<(), Error> {
+    if value == KeyValue::Press {
+        ktrl.l_mgr.toggle_profile(name, on)
+    }
+
+    Ok(())
+}
+
+fn perform_toggle_all_profiles(ktrl: &mut Ktrl, value: KeyValue) -> Result<(), Error> {
```

Comment:
I'd rename this to `perform_deactivate_all_profiles` :)

Reply:
Yep sounds good